src/js/tabs/setup.js

```diff
@@ -237,6 +237,7 @@ TABS.setup.initialize = function (callback) {
             if (semver.gte(CONFIG.apiVersion, "1.42.0")) {
                 disarmFlagElements.splice(disarmFlagElements.indexOf('THROTTLE'), 0, 'CRASH');
                 disarmFlagElements = disarmFlagElements.concat(['REBOOT_REQD']);
+                disarmFlagElements = disarmFlagElements.concat(['FILTERING']);
```

Comment:
I think remember the concat method can receive more than one element, so is better for a cleaner code to add it to the earlier concat with appropiate formatting of code.